e2e/playwright/folder-rules/src/tests/manage-rules.spec.ts

```diff
@@ -22,27 +22,29 @@
  * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { getUserState, test } from '@alfresco/playwright-shared';
+import { ApiClientFactory, getUserState, test } from '@alfresco/playwright-shared';
 
-test.use({ storageState: getUserState('admin') });
+test.use({ storageState: getUserState('hruser') });
```

Comment:
Why do you need hruser instead of admin?

Reply:
Should we use admin for the job that can be done by normal hruser ? Should we use admin user for not admin app at all?